src/OpenLoco/src/Objects/RoadObject.h

```diff
@@ -16,42 +17,46 @@ namespace OpenLoco
         struct RenderTarget;
     }
 
-    namespace Flags12
+    enum class Flags12 : uint16_t // Made this a uint16_t to maintain the size of the RoadObject struct
```

Comment:
Okay, makes sense. Could you rename flags 12 to road object flags. Perhaps also be an idea to rename the piece flags to add object into the name as well as it may conflict eventually with something else